packages/viewer/src/components/viewer/viewer.tsx

```diff
@@ -861,9 +861,10 @@ export class Viewer {
    *  * `urn:vertex:scene:<sceneid>`
    *
    * @param urn The URN of the resource to load.
+   * @param cameraType (Optional) The camera type to load. If not included, the default camera type for the resource will be used.
    */
   @Method()
-  public async load(urn: string): Promise<void> {
+  public async load(urn: string, cameraType?: FrameCameraType): Promise<void> {
```

Comment:
One thought here - could we model this as a set of options versus a singular parameter? Thinking something along the lines of what we do for `applySceneViewState` or `resetView`. I could definitely see the `load` method expanding to support additional "initialization" features down the  road

Reply:
Good point - updated!